torch/testing/_internal/distributed/nn/api/remote_module_test.py

```diff
@@ -17,6 +18,7 @@
 
 _PARAM_VAL = torch.nn.Parameter(torch.ones(1))
 
+TEST_WITH_ROCM = os.getenv('PYTORCH_TEST_WITH_ROCM', '0') == '1'
```

Comment:
Please reuse the variable as follows: `from torch.testing._internal.common_utils import TEST_WITH_ROCM`

Reply:
done